go-controller/pkg/cluster/node.go

```diff
@@ -29,21 +29,19 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 	var subnet *net.IPNet
 
 	for count > 0 {
-		if count != 30 {
-			time.Sleep(time.Second)
-		}
-		count--
-
 		// setup the node, create the logical switch
 		node, err = cluster.Kube.GetNode(name)
 		if err != nil {
 			logrus.Errorf("Error starting node %s, no node found - %v", name, err)
+			time.Sleep(time.Second)
```

Comment:
May be I will just increase the counter from 30 second to 300 seconds then. I guess, waiting for 5 min is fine. The worry is that since we are doing node-init in master too, we may fail there and exit resulting in net-controller not starting.